lib/models/node.js

```diff
@@ -64,6 +69,31 @@ function NodeModelFactory (Model, waterline, _) {
                     return !_.isEmpty(catalog);
                 });
             }
+        },
+        addTags: function(id, tags) {
+            return Promise.fromNode(this.native.bind(this))
+                .then(function(collection) {
+                    return collection.update(
+                        { _id: waterline.nodes.mongo.objectId(id) },
+                        { 
+                            $addToSet: { tags: { $each: tags } },
+                            $set: { updatedAt: new Date() }
+                        });
+                });
+        },
+        remTags: function(id, tag) {
+            return Promise.fromNode(this.native.bind(this))
+                .then(function(collection) {
+                    return collection.update(
```

Comment:
I'm not sure whether @benbp  have concerns about introducing mongodb native commands to RackHD compatibility in the future. 

it's ok for me, because waterline CRUD cannot cover the cases such as remove a key:value in one object here.
Just think that it's better to move mongodb native commands to some common place so that it could be leveraged by other codes, just like removing a 'key:value' here is a common case. and previous waterline CRUD message is published to amqp 
https://github.com/RackHD/on-core/blob/master/lib/common/model.js#L62-L89 , the native operations in a common place could also so similar things